app/controllers/test_executions_controller.rb

```diff
@@ -82,7 +83,7 @@ def set_product_test_from_task
   def add_breadcrumbs
     add_breadcrumb 'Dashboard', :vendors_path
     add_breadcrumb 'Vendor: ' + @product_test.product.vendor.name, vendor_path(@product_test.product.vendor)
-    add_breadcrumb 'Product: ' + @product_test.product.name, vendor_product_path(@product_test.product.vendor, @product_test.product)
+    add_breadcrumb 'Product: ' + @product_test.product_name, vendor_product_path(@product_test.product.vendor, @product_test.product)
```

Comment:
There's still a lot of trainwrecking in the method. I'm not sure the best way to solve it, but I think that's what making things so unreadable.

Reply:
I'm gonna push a first round of changes before I fix this, just because it's still unsure how to go about the fix. 

I will start by changing the above line too though, since it's strange that rails_best_practices didn't call it out. It will at least make it a bit more consistent.

Reply:
Actually, I'm going to try substituting local variables to make it more readable.